src/firebase.js

```diff
@@ -0,0 +1,13 @@
+import { initializeApp } from "firebase/app";
+import { getFirestore } from "firebase/firestore";
+import { getAuth } from "firebase/auth";
+import firebaseConfig from "firebaseConfig.js";
+
+//  acts as entire point between firebase and React app
+const firebaseApp = initializeApp(firebaseConfig);
+
+const auth = getAuth(firebaseApp);
+
+const db = getFirestore(firebaseApp);
+
+export { auth, db, firebaseApp };
```

Comment:
I think firebaseApp is not required to import

Reply:
Yeah,Thanks for pointing it out Nisha